src/components/application_manager/src/resumption/resume_ctrl_impl.cc

```diff
@@ -657,13 +657,15 @@ void ResumeCtrlImpl::StartWaitingForDisplayCapabilitiesUpdate(
   smart_objects::SmartObject saved_app(smart_objects::SmartType_Map);
   resumption_storage_->GetSavedApplication(
       application->policy_app_id(), application->mac_address(), saved_app);
+  auto& builder = application->display_capabilities_builder();
   auto resume_callback =
-      [this](Application& app,
-             const smart_objects::SmartObject& display_capabilities) -> void {
+      [&builder, this](
```

Comment:
@LitvinenkoIra you have an `app` passed to that lambda so you may get builder inside of that function instead of passing ref from outside

Reply:
@AKalinich-Luxoft Done https://github.com/LuxoftSDL/sdl_core/commit/f8f66c7c6ad7da8f2ac191942531918282b663df#diff-89664857793adc72d087a6958999865d16e2ee4c595fb4c23fa4e397bad4e93eR665